Documentation/Contributing/DevDocGuide.md

```diff
@@ -63,6 +63,33 @@ This links to the [BoundarySystem API](xref:Microsoft.MixedReality.Toolkit.Bound
 as well as this short version: @Microsoft.MixedReality.Toolkit.Boundary
 ```
 
+## Enumerating available xrefs
+
+xref syntax can be difficult to remember - it's possible to enumerate all of the available xref IDs by first running
```

Comment:
```suggestion
Xref syntax can be difficult to remember - it's possible to enumerate all of the available xref IDs by first running
```

Xref looks weird but it's the beginning of a sentence so it should be uppercase I think?